docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -89,28 +91,36 @@ export default class MasonryContainer extends Component<Props<{ ... }>, State> {
   componentDidMount() {
     window.TEST_FETCH_COUNTS = 0;
 
-    window.addEventListener('trigger-reflow', () => {
-      if (this.gridRef.current) {
-        this.gridRef.current.reflow();
-        this.forceUpdate();
-      }
-    });
+    if (!eventsSet) {
```

Comment:
instead of doing this, could we just remove the event listener on unmount?

```
componentDidMount() {
  window.addEventListener('trigger-reflow')...
}
componentWilllUnmount() {
  window.removeEventListener('trigger-reflow')...

```

still unexpected that this is happening twice though